frontend/packages/policy-editor/src/utils/index.ts

```diff
@@ -44,7 +44,7 @@ export const mapPolicySubjectToSubjectTitle = (
 const findSubjectOptionBySubjectId = (
   subjectOptions: PolicySubject[],
   subjectId: string,
-): PolicySubject => subjectOptions.find((s) => s.subjectId === subjectId);
+): PolicySubject => subjectOptions.find((s) => s.subjectId === subjectId.trim()); // .trim() is added to make sure there is no additional whitespace on the ID before it's being checked.
```

Comment:
sounds wired to do this, but this is maybe a temp fix to actually find out why the whitespace is there in the first place?

Reply:
Totally agree with you. This should ideally not be done in frontend at all, but for some reason this is the only way to fix the bug today 🙁 I'll try to remove it at a later time 😄